apps/frontend/src/auth.ts

```diff
@@ -9,22 +10,77 @@ declare module '@auth/sveltekit' {
 	}
 }
 
+const clientId = env.AUTH_DESCOPE_ID;
+const clientSecret = env.AUTH_DESCOPE_SECRET;
+
+if (!clientId || !clientSecret) {
+	throw new Error('Missing Descope client credentials');
+}
```

Comment:
This would execute during build and -- hence -- require these variables to be set during.

Instead I would suggest we check them right before we use them.